src/NEARWEEK/News.jsx

```diff
@@ -221,6 +226,8 @@ function dateToDays(date) {
   return timeAgo(Date.now() - d.getTime());
 }
 
+console.log(news);
```

Comment:
Is it intentional?

Reply:
no, its not necessary, removed here https://github.com/near/near-discovery-components/pull/120/commits/013dad164b1385cdee72733e5adb36cf8b7fc5d0 thanks for checking